developer-docs-site/docs/guides/move-guides/mint-nft-cli.md

```diff
@@ -3,76 +3,428 @@ title: "Mint an NFT with Aptos CLI"
-testnet
+aptos move publish --named-addresses mint_nft=<default-account-address>
 ```
 
-1. When prompted for your private key, hit enter to generate a new key:
+For example:
+
 ```shell
-Enter your private key as a hex literal (0x...) [Current: None | No input: Generate new key (or keep one if present)]
+aptos move publish --named-addresses mint_nft=a911e7374107ad434bbc5369289cf5855c3b1a2938a6bfce0776c1d296271cde
 ```
 
-1. Receive output resembling:
+3. Receive output asking you to accept a gas fee for the transaction:
+
 ```shell
-No key given, generating key...
-Account blah does not exist, you will need to create and fun the account through a community faucet e.g. https://aptoslabs.com/testnet-faucet, or by transferring funds from another account
+    Compiling, may take a little while to download git dependencies...
+    INCLUDING DEPENDENCY AptosFramework
+    INCLUDING DEPENDENCY AptosStdlib
+    INCLUDING DEPENDENCY AptosToken
+    INCLUDING DEPENDENCY MoveStdlib
+    BUILDING Examples
+    package size 2770 bytes
+    Do you want to submit a transaction for a range of [1164400 - 1746600] Octas at a gas unit price of 100 Octas? [yes/no]
+```
 
----
-Aptos CLI is now set up for account blah as profile nft-receiver!  Run `aptos --help` for more information about commands
-{
-  "Result": "Success"
-}
-➜  devel
+4. Enter `yes` and receive results indicating success:
+
+```shell
+    {
+      "Result": {
+        "transaction_hash": "0x576a2e9481e71b629335b98ea75c87d124e1b435e843e7a2ef8938ae21bebfa3",
+        "gas_used": 11679,
+        "gas_unit_price": 100,  
+        "sender": "a911e7374107ad434bbc5369289cf5855c3b1a2938a6bfce0776c1d296271cde",
+        "sequence_number": 0,
+        "success": true,
+        "timestamp_us": 1669659103283876,
+        "version": 12735152,
+        "vm_status": "Executed successfully"
+      }
+    }
+```
+
+### See the module on Aptos Explorer
+
+1. Go to the [Aptos Explorer](https://explorer.aptoslabs.com/) in a web browser>
+2. Select the network you used: testnet
+3. Search for the transaction by `transaction_hash` in the search field, using your own unique transaction hash.
+4. View the changes made in publishing this module under the *Changes* tab.
+
+## Mint a token to a receiver
+
+See [create_nft.move](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/move-examples/mint_nft/1-Create-NFT/sources/create_nft.move) to see how to mint a token to a receiver.
+
+## 2. Use resource account for automation
+
+Since the Move model often requires knowing the signer of a transaction, Aptos provides resource accounts for assigning signer capability. Creating resource accounts enables access to the signer capability for automated use. With this ability, resource accounts can publish Move modules and automatically sign for transactions and minting. The signer capability can be assigned to the resource account or placed in storage locally within the module.
```

Comment:
`The signer capability can be assigned to the resource account`

could you clarify what this means?

Reply:
Ooh I see, in this case I think maybe "the signer capability can be retrieved by the resource account's signer in combination with the address of the source account which created the resource account" is more clear. 

`        let resource_signer_cap = resource_account::retrieve_resource_account_cap(resource_signer, @source_addr);
`
^ this line of code retrieves the signer: https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/move-examples/mint_nft/2-Using-Resource-Account/sources/create_nft_with_resource_account.move#L136.
would be great for educational purpose if we can link it in the tutorial

Reply:
Done! Although this text is now going in the Concepts section of:
https://aptos.dev/guides/move-guides/move-on-aptos/#key-concepts-in-move